src/krb5_plugin/passkey/passkey_utils.c

```diff
@@ -535,7 +535,7 @@ sss_passkey_prefix_json_data(enum sss_passkey_phase phase,
     json_error_t jret;
```

Comment:
The input string to sss_passkey_prefix_json_data is
~~~
 {"credential_id": "xlAsvaMlLJkCp74uRWDamrVCSd2+Y1SC4cIDNmwT6DQhuC/51Lp5lSQU9tQmWl8DQzrb0XBPRJ41CaZ5TKnKRw==", "cryptographic_challenge": "0Ny+OeU9309V839tjcG2miZrOcpA40L8ohfyaRK02P0=", "authenticator_data": "WCWcAO58pq6rRNqWEXg/P4ZkES9OqDS8L29D9KqffS71WwUAAAAE", "assertion_signature": "MEQCIGsirLnyMMemr9gcxZDjL82Z4YzwqlqiruRvyorjHbBWAiA1ST2fqPgk4hQfS/1VQHjqKjRHWt85K1ya5VZE5WjDWA==", "user_id": ""}
~~~
It does not contain the `passkey {phase...state...` fields, is there a better way to handle this?

Reply:
Ah, I see the difference now. Maybe it would be good to rename it to `sss_passkey_message_from_reply_json` or something like that, because "passkey" prefix is not needed/added and it only works with reply data.

Or add full message format in `sss_authtok_set_passkey_reply`  then you can use `sss_passkey_message_from_json` directly.

Reply:
But this is not relevant to this PR.

Reply:
In that case, and if you don't have any other comment, please approve.

Reply:
I re-run covscan and it finished successfully now. Ack.

Reply:
@ikerexxe could you take care of renaming `sss_passkey_prefix_json_data` to `sss_passkey_message_from_reply_json` as Pavel suggests as part of this PR?

Reply:
Done in a new commit